x-pack/platform/test/api_integration/services/ml/api.ts

```diff
@@ -1591,6 +1591,16 @@ export function MachineLearningAPIProvider({ getService }: FtrProviderContext) {
       }
     },
 
+    async startTrainedModelDeploymentES(modelId: string) {
+      log.debug(`Starting trained model deployment with id "${modelId}"`);
+      const url = `/_ml/trained_models/${modelId}/deployment/_start`;
+
+      const { body, status } = await esSupertest.post(url);
+      this.assertResponseStatusCode(200, status, body);
+
+      log.debug('> Trained model deployment started');
+    },
```

Comment:
It was added to the `ml` service under `x-pack/test/functional`, copying it for compatibility. The one in `x-pack/test/functional` will be removed later when we relocate functional tests.